INSTALL.md

```diff
@@ -1,98 +1,107 @@
 # Installation of Clusive
 
-Clusive can be installed as a local server for development purposes,
-or via Docker for production use.
+Clusive can be installed locally for development or for production use via Docker.
+
 
 ## Local Installation
 
 ### Prerequisites
 
 * [Python 3](https://www.python.org/downloads/)
-  * On Mac, Homebrew is the easiest way to do this (OS X comes with Python 2.7); 
-    you'll need to use the *python3* and *pip3* commands to distinguish Python 2 and Python 3, 
-    use virtualenv, or change your aliases.
-* [virtualenv](https://virtualenv.pypa.io/en/stable/installation/) 
-  * Not required but strongly recommended for maintaining an isolated environment and dependencies.
+  * Version 3.7.4 or later. On Mac, Homebrew is the easiest way to install
+* [virtualenv](https://virtualenv.pypa.io/en/latest/) 
+  * Highly recommended for maintaining an isolated environment and dependencies
 * [Postgres](https://www.postgresql.org/) 
-  * Used in the deployment configuration; 
-    may also be necessary for the psycopg2 dependency (Homebrew on OS X can take care of this)
-* [npm](https://www.npmjs.com/)
+  * Version 11.5 or later. Used in the deployment configuration 
+* [npm/grunt](https://www.npmjs.com/get-npm) 
   * Used as a build system to manage all other dependencies
```

Comment:
```suggestion
* [Node.js](https://nodejs.org/)
  * Used for the build system and to manage all other dependencies using:
    * [npm](https://www.npmjs.com/get-npm) - Package manager
    * [Grunt](https://gruntjs.com/) - Task runner
```

Reply:
If I understand correctly, npm is the prerequisite that you need to install manually.  You should get grunt installed for you when you type 'npm install'.

Reply:
- npm is installed when Node.js is installed.
- grunt is installed by `npm install`